### DISCUSSION TITLE
Additional options question

### DISCUSSION THREAD

**Discussion by ukrolelo** [postalserver/postal#1598] [Help with using Postal] [answered]
Greetings,
i have some experience with kerio connect.
I am planing to migrate to Postal,now i have the docker installation. How its possible to:
1.See security log when somebody is trying to brute force password
2.Spam logs
3.How to limit users to only be able to connect from local lan
4.Where can i set DNS black list like Spamhaus SpamCop etc.
5.Is therer any custom rules like if this domain or this body then send to spam
6.Does it have attachment filter? Like dont allow exe files?
Thankyou

**Comment by willpower232** [comment, marked as answer]
1. I don't think this is a feature but I wonder what might be available in ruby on rails
2. I don't think this is a feature, each message is quite separated, there isn't centralised logging for all messages
3. you would need to do this in your own firewall
4. I don't think this is a feature, Postal handles all messages it is given
5. that is not a feature of Postal but you can configure spamassassin or rspamd so you might be able to accomplish it with them
6. Postals job is not to block attachments, you should block this when sending messages from your application
